pkg/auth/providers/saml/saml_provider.go

```diff
@@ -120,6 +145,52 @@ func PerformSamlLogin(name string, apiContext *types.APIContext, input interface
 	return nil
 }
 
+func (s *Provider) getLDAPConfig() (*v3.LdapConfig, *x509.CertPool, error) {
```

Comment:
Is this code just all copy and pasted from somewhere else? If yes, why did you need to do that rather than reuse the existing function (if there is no easy way to reuse, thats fine, just want to know).